libs/utils/analysis/latency_analysis.py

```diff
@@ -387,9 +387,11 @@ def _taskState(self, state):
             256: "W", # TASK_WAKING
             512: "P", # TASK_PARKED
            1024: "N", # TASK_NOLOAD
-           2048: "n", # TASK_NEW
         }
-        TASK_MAX_STATE = 4096
+        kver = float(self._trace.platform.get('kernel_version', '3.18'))
+        if kver > 4.7:
```

Comment:
If @bjackman adds the new __cmp__ API to devlib.... this should be updated as well...